docs/data/system/experimental-api/extend-sx-prop/extend-sx-prop.md

```diff
@@ -0,0 +1,24 @@
+# Extend the `sx` prop
+
+<p class="description">Learn about the experimental API for extending or changing the behavior of the `sx` prop.</p>
+
+## Extend the `sx` prop
+
+If you wish to add new keys that would be processed by the `sx` prop, you can do that by extending the `unstable_sxConfig` option inside the theme.
+
+{{"demo": "ExtendTheSxProp.js" }}
+
+## Override some of the existing behavior
+
+There is also a possibility to change some of the existing behavior. You can simply provide a custom config in the for the specific key.
+
+{{"demo": "ChangeTheBehaviorSxProp.js" }}
+
```

Comment:
A demo with dynamic computation like in https://stitches.dev/docs/utils could be great. Taking https://github.com/mui/material-ui/issues/32734#issuecomment-1317538857, it could look like this:`<Box paddingX="medium" />`, or `size`, a property for applying width/height together, we don't have it in the system by default.

Reply:
Will add the size option 👍